doc/nrf/releases/release-notes-1.9.1.rst

```diff
@@ -0,0 +1,99 @@
+The feature is made available in its current state though the design and interfaces can change between release tags.
+The feature will also be labelled with "EXPERIMENTAL" in Kconfig files to indicate this status.
+Build warnings will be generated to indicate when features labelled EXPERIMENTAL are included in builds unless the Kconfig option :kconfig:`CONFIG_WARN_EXPERIMENTAL` is disabled.
+
+Highlights
+**********
+
+* Entry 1
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v1.9.1**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions`_.
+
+IDE support
+***********
+
+|NCS| v1.9.1 is supported by both `nRF Connect extensions for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ and :ref:`installing_ses`.
+In future releases, documentation on the setup and usage of Segger Embedded Studio Nordic Edition will no longer be available and all references to IDE support will instruct users with nRF Connect extensions for Visual Studio Code.
+In future releases, Segger Embedded Studio will no longer be tested with the build system and it is not recommended for new projects.
+
+nRF Connect extensions for Visual Studio Code will be under continued active development and supported and tested with future |NCS| releases.
+It is recommended for all new projects.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - :ref:`nrfxlib:nrf_modem_changelog`
+   * - LwM2M carrier library
+     - :ref:`liblwm2m_carrier_changelog`
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+
+.. WIP: PLACEHOLDER
+
+  See `known issues for nRF Connect SDK v1.9.0`_ for the list of issues valid for the latest release.
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Trusted Firmware-M
+==================
+
+* Fixed an issue that would cause crash during boot when the RPMsg was enabled on the nRF5340 DK.
+
```

Comment:
should probably add the MDK FICR flash copy fix, yeah @joerchan  ?

Reply:
Yes, will do that once it is approved. Included in this PR: https://github.com/nrfconnect/sdk-nrf/pull/6947